lib/matplotlib/rcsetup.py

```diff
@@ -1117,6 +1131,7 @@ def _validate_linestyle(ls):
     'mathtext.fontset':        ['dejavusans', validate_fontset],
     'mathtext.default':        ['it', validate_mathtext_default],
     'mathtext.fallback_to_cm': [True, validate_bool],
+    'mathtext.fallback':       [None, validate_mathtext_fallback],
```

Comment:
This needs some documentation.  In particular it seems `fallback` overrides `fallback_to_cm`  if its set; there should be a comment in here.  

Preferably, we would deprecate `fallback_to_cm` and make `cm` the default for `mathtext.fallback`.  

This needs to be in `matplotlibrc.template` as well, right?